pdr_backend/aimodel/test/test_aimodel_general.py

```diff
@@ -390,8 +390,8 @@ def test_aimodel__regr_0error__via_10000x(approach):
         calibrate_probs="None",
         calibrate_regr="None",  # explicitly None here
     )
-    for curprice_xval in [-8, -5, -2, 2, 5, 8]:
```

Comment:
Don't change the names in this file. Why: this file is more general, it doesn't matter, let's keep the PR small.